x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -67,192 +80,201 @@ export function AddDataView() {
+            ),
+          },
+          {
+            title: i18n.translate('xpack.profiling.tabs.kubernetesValidationStep', {
+              defaultMessage: 'Validate the host-agent pods are running:',
+            }),
+            content: (
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {i18n.translate(
+                  'xpack.profiling.tabs.kubectl-nUniversal-profilingGetCodeBlockLabel',
+                  { defaultMessage: 'kubectl -n universal-profiling get pods' }
+                )}
+              </EuiCodeBlock>
+            ),
+          },
+          {
+            title: i18n.translate('xpack.profiling.tabs.postValidationStep', {
+              defaultMessage:
+                'Use the Helm install output to get host-agent logs and spot potential errors',
+            }),
+            content: <></>,
+          },
+        ],
+      },
+      {
+        key: AddDataTabs.Docker,
+        title: i18n.translate('xpack.profiling.tabs.dockerTitle', {
+          defaultMessage: 'Docker',
+        }),
+        steps: [
+          {
+            title: i18n.translate('xpack.profiling.tabs.dockerRunContainerStep', {
+              defaultMessage: 'Run the Universal Profiling container:',
+            }),
+            content: (
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {`docker run --name pf-host-agent --privileged --pid=host -v /etc/machine-id:/etc/machine-id:ro \\
 -v /var/run/docker.sock:/var/run/docker.sock -v /sys/kernel/debug:/sys/kernel/debug:ro \\
 docker.elastic.co/observability/profiling-agent:${stackVersion} /root/pf-host-agent \\
 -project-id=1 -secret-token=${secretToken} \\
 -collection-agent=${collectionAgentHost}`}
-            </EuiCodeBlock>
-          ),
-        },
-      ],
-    },
-    {
-      key: AddDataTabs.Binary,
-      title: i18n.translate('xpack.profiling.tabs.binaryTitle', {
-        defaultMessage: 'Binary',
-      }),
-      steps: [
-        {
-          title: i18n.translate('xpack.profiling.tabs.binaryDownloadStep', {
-            defaultMessage: 'Download the binary for the right architecture:',
-          }),
-          content: (
-            <EuiText>
-              <b>For x86_64:</b>
-              <EuiSpacer size="s" />
-              <EuiCodeBlock paddingSize="s" isCopyable>
-                {`wget -O pf-host-agent.tgz "https://artifacts.elastic.co/downloads/prodfiler/pf-host-agent-${stackVersion}-linux-x86_64.tar.gz" && tar xzf pf-host-agent.tgz`}
               </EuiCodeBlock>
-              <EuiSpacer size="m" />
-              <b>For ARM64:</b>
-              <EuiSpacer size="s" />
+            ),
+          },
+        ],
+      },
+      {
+        key: AddDataTabs.Binary,
+        title: i18n.translate('xpack.profiling.tabs.binaryTitle', {
+          defaultMessage: 'Binary',
+        }),
+        // Create a dedicated sub tab for each architecture that we support
+        subTabs: ['x86_64', 'arm64'].map((arch: string) => {
```

Comment:
No need to type `arch` here as it'll be inferred by TS.
```suggestion
        subTabs: architectures.map((arch) => {
```
* See comment below 👇🏻